examples/examples_dotnet_test.go

```diff
@@ -129,6 +134,36 @@ func TestSecretsInExplicitProvider(t *testing.T) {
 		Dir:         path.Join(getCwd(t), "test-secrets-in-explicit-provider", "csharp"),
 		Quick:       true,
 		SkipRefresh: true,
+
+		ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+			deploymentJSON, err := json.MarshalIndent(stack.Deployment, "", "  ")
+			assert.NoError(t, err)
+
+			t.Run("program-initiated secrets are encrypted", func(t *testing.T) {
+				t.Skip("TODO <ticket>")
```

Comment:
This is still a problem. All these secrets leak into the statefile. We need to ticket and find a fix strategy, unclear if this should go back to the bridge or we can fix the .NET SDK gen; might need to think this through and cross-test with TypeScript SDK gen.